src/Compilers/CSharp/Portable/Binder/PatternSwitchBinder2.cs

```diff
@@ -231,4 +239,57 @@ internal override void BindPatternSwitchLabelForInference(CasePatternSwitchLabel
             }
         }
     }
+
+    partial class BoundPatternSwitchStatement2
```

Comment:
>BoundPatternSwitchStatement2 [](start = 18, length = 28)

This code should be moved into a file under BoundTree folder.

Reply:
Addressed in https://github.com/dotnet/roslyn/pull/24059/commits/114a56e9077ed03f08443889b69cae6972a4acc6 of https://github.com/dotnet/roslyn/pull/24059